internal/docker/builder.go

```diff
@@ -180,17 +180,32 @@ func (d *Builder) ConstructBlueprintIfNotExist(bprint b.Blueprint) error {
 	if err != nil {
 		return fmt.Errorf("ConstructBlueprintIfNotExist(%s): failed to ImageList: %w", bprint.Name, err)
 	}
-	if len(images) == 0 {
-		err = d.ConstructBlueprint(bprint)
+	var missingHomeservers []b.Homeserver
```

Comment:
There are many ways to tackle this sort of behavior. What's the most idiomatic Complement way to accomplish this? Different argument name?

Reply:
This other comment is perfect, https://github.com/matrix-org/complement/pull/454#discussion_r975202948

Just looking for a way forward as I know the the quick and dirty way here wasn't good enough to merge.